junit-jupiter-api/src/main/java/org/junit/jupiter/api/DisplayNameGenerator.java

```diff
@@ -381,28 +384,31 @@ private String getSentenceBeginning(List<Class<?>> enclosingInstanceTypes, Class
 		 * will be used.
 		 *
 		 * @param testClass the test class to search on for {@code @IndicativeSentencesGeneration}
+		 * @param enclosingInstanceTypes the runtime types of the enclosing
+		 * instances; never {@code null}
 		 * @return the sentence fragment separator
 		 */
-		private static String getFragmentSeparator(Class<?> testClass) {
-			return findIndicativeSentencesGeneration(testClass)//
+		private static String getFragmentSeparator(Class<?> testClass, List<Class<?>> enclosingInstanceTypes) {
+			return findIndicativeSentencesGeneration(testClass, enclosingInstanceTypes)//
 					.map(IndicativeSentencesGeneration::separator)//
 					.orElse(IndicativeSentencesGeneration.DEFAULT_SEPARATOR);
 		}
 
 		/**
 		 * Get the display name generator to use for the supplied test class.
-		 *
 		 * <p>If {@link IndicativeSentencesGeneration @IndicativeSentencesGeneration}
```

Comment:
Not sure why you deleted that line. 🤔

Reply:
Me either. :slightly_smiling_face: 

Restored in 52fb535c8fdb58b7e8e089c9b655989840827eda.